src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/StatusBar/StatusBarBehavior.shared.cs

```diff
@@ -57,8 +57,31 @@ protected override void OnAttachedTo(Page bindable, object platformView)
 	{
 		StatusBar.SetColor(StatusBarColor);
 		StatusBar.SetStyle(StatusBarStyle);
+#if IOS
+		bindable.SizeChanged += new EventHandler(page_SizeChanged);
+#endif
+	}
+
+	/// <inheritdoc /> 
+#if IOS
+	protected override void OnDetachedFrom(Page bindable, UIKit.UIView platformView)
+#elif ANDROID
```

Comment:
Do we need other platforms if the code inside the method is only for iOS

Reply:
@VladislavAntonyuk, Since the method definition for each platform other than iOS is no longer necessary, delete it.

Reply:
I will upload it again. OnDetachedFrom is unnecessary except for iOS.